inventory.py

```diff
@@ -93,57 +99,109 @@ def gather_course_info_for_account(account_id: int, term_id: int) -> Sequence[in
             more_pages = False
 
     course_df = pd.DataFrame(slim_course_dicts)
-    course_df['course_warehouse_id'] = course_df['course_id'].map(lambda x: x + WAREHOUSE_INCREMENT)
+    course_df['warehouse_id'] = course_df['canvas_id'].map(lambda x: x + WAREHOUSE_INCREMENT)
     logger.debug(course_df.head())
-    course_df.to_csv(os.path.join('data', 'course.csv'), index=False)
-    logger.info('Course data was written to data/course.csv')
-    course_ids = course_df['course_warehouse_id'].to_list()
-    return course_ids
+    return course_df
 
 
-def pull_enrollment_and_user_data(udw_course_ids) -> None:
-    udw_courses_string = ','.join([str(udw_course_id) for udw_course_id in udw_course_ids])
+def pull_enrollment_data_from_udw(course_ids) -> pd.DataFrame:
+    courses_string = ','.join([str(course_id) for course_id in course_ids])
     enrollment_query = f'''
-        SELECT e.id AS enrollment_id,
-               e.canvas_id AS enrollment_canvas_id,
-               e.user_id AS enrollment_user_id,
-               e.course_section_id AS enrollment_course_section_id,
-               e.course_id AS enrollment_course_id,
-               e.workflow_state AS enrollment_workflow_state,
+        SELECT e.id AS warehouse_id,
```

Comment:
we may only want to select student enrollment with e.workflow_state='active'

Reply:
Let's check with @jennlove-um about this.

Reply:
Selecting onlyl "active" students should be okay.  The only other state that I was considering was "invited," but until students accept the invitation, they aren't really enrolled.  So let's go with workflow_state='active' as suggested.

Reply:
I'm seeing these values in the enrollment table for `workflow_state`.
```| active           |
| deleted          |
| completed        |
| invited          |
| inactive         |
| rejected         |
| creation_pending```

Reply:
Which of these should we keep?

Reply:
We should keep 'active' because for this report we only care about students who are actively enrolled in the course.  None of those other statuses should matter for this.

Reply:
Okay, I will try this out.

Reply:
I am now filtering on the 'active' `workflow_state`, as of [this commit](https://github.com/tl-its-umich-edu/course-inventory/pull/10/commits/1f312a45da35623f71e54a63c0566b8a34447717).